aptos-move/move-examples/post_mint_reveal_nft/Move.toml

```diff
@@ -9,4 +9,4 @@ AptosToken = { local = "../../framework/aptos-token" }
 DataStructure = { local = "../data_structures" }
 
 [addresses]
-source_addr = "0xcafe"
+source_addr = "_"
```

Comment:
This is intentional?

Reply:
This should be handled in this PR https://github.com/aptos-labs/aptos-core/pull/5955

Reply:
Yes https://github.com/aptos-labs/aptos-core/pull/5955/files/cd5196cc860e0d3646a1058218703b28a8b1fd15..bcd4a50a202504073d0748590ea09b63f488fd6e has more context

Reply:
Then please rebase on PR https://github.com/aptos-labs/aptos-core/pull/5955. Thanks for clarifying